package.json

```diff
@@ -156,7 +156,7 @@
     "postcss-styled-syntax": "^0.6.4",
     "postcss-syntax": "^0.36.2",
     "prettier": "^3.2.4",
-    "puppeteer": "^21.9.0",
+    "puppeteer": "^21.11.0",
```

Comment:
I was thinking updating to the latest _major_ version of Puppeteer, not just bumping the minor.

Interestingly though, that seems to be causing the e2e failures I saw recently in #13612

#13675 also causes test failures. That's not great 😞